### PR TITLE
fix(#514): simplify public-facing types by inlining type-aliases

### LINKED ISSUE #514: fix: simplify public-facing types by inlining type-aliases
E.g. `init_df` of `ValueFrame|TimestampValueFrame` takes an `InitDF_T` which is hard to parse without a doc string specifying it or by changing the type to be human readable. 

Others:

- [x] look[behind|ahead] distance: `LookDistances` (sequence of dt.timedelta)
- [x] fallback: ValueType 
- [x] entity_id_col_name: default_entity_id_col_name (instead of the string repr)
- [x] default_pred_time_col_name
- [x] default_prediction_time_uuid_col_name
- [x] default_timestamp_col_name

### PR DESCRIPTION
**Pull request by HLasse**
fix(#514): simplify public-facing types by inlining type-aliases

Fixes #514

refactor: inline LookDistance and LookDistances

refactor: inline ValueType

refactor: inline default entity_id_col_name

refactor: inline default_prediction_time_uuid_col_name

refactor: inline default_pred_time_col_name

refactor: inline default_timestamp_col_name

refactor: inline InitDF_T

lint

### CODE REVIEW

**Comment by HLasse** [comment]
* **#564** <a href="https://app.graphite.dev/github/pr/Aarhus-Psychiatry-Research/timeseriesflattener/564?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @HLasse and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by MartinBernstorff — APPROVED** [review]
I'll let the judgement be up to you 👍

**Comment by MartinBernstorff** [comment]
@HLasse Ready to make a decision here? 😉

**Comment by HLasse** [comment]
On the list for today 👍
